modules/terraform/init.go

```diff
@@ -27,6 +27,10 @@ func InitE(t testing.TestingT, options *Options) (string, error) {
 	if options.MigrateState {
 		args = append(args, "-migrate-state", "-force-copy")
 	}
+	// Append no-color option if needed
+	if options.NoColor {
```

Comment:
Will be helpful to have tests which will track that `NoColor` flag will continue to work over time